frontend/src/routes/Applications/ApplicationsPage.tsx

```diff
@@ -41,12 +68,10 @@ export default function ApplicationsPage() {
                     title={t('Applications')}
                     navigation={
                         <AcmSecondaryNav>
-                            <AcmSecondaryNavItem isActive={location.pathname.endsWith(NavigationPath.applications)}>
+                            <AcmSecondaryNavItem isActive={applicationsMatch.isExact}>
```

Comment:
Horizontal nav **Overview** tab was not highlighted if you navigated to `/multicloud/applications/` with a trailing slash. `isExact` allows for the trailing slash.